content/tutorials/xgboost_deployment.md

```diff
@@ -101,8 +79,6 @@ Now your application is ready to serve predictions. Let's try one!
 ...         print('Prediction Returned:', result)
 ...
 ('Prediction Returned:', {u'default': False, u'output': [0.5], u'query_id': 26})
```

Comment:
Put the output in a separate code block, and say something like "You should see something like the following output."